cvat/apps/engine/models.py

```diff
@@ -181,6 +183,242 @@ def choices(cls):
     def __str__(self):
         return self.value
 
+
+T = TypeVar("T")
+
+
+def _parse_both_before_accessing(fn):
```

Comment:
Please give a more clear name to the function. It's not clear, what this "both" is.

Reply:
Done